gerrymandering/gerrymandering.ipynb

```diff
@@ -116,17 +110,29 @@
    "outputs": [],
    "source": [
     "x_range, y_range = ((-13884029.0, -7453303.5), (2818291.5, 6335972.0)) # Continental USA\n",
-    "shade_defaults = dict(x_range=x_range, y_range=y_range, x_sampling=10, y_sampling=10, width=1200, height=682,\n",
-    "                      color_key=color_key, aggregator=ds.count_cat('race'),)\n",
-    "shaded = datashade(census_points, **shade_defaults)\n",
+    "\n",
+    "shaded = df.hvplot.points(\n",
+    "    'easting', 'northing',\n",
+    "    datashade=True,\n",
```

Comment:
Per the checklist:

```suggestion
    "    rasterize=True,\n",
```

Reply:
rasterize doesn't work yet for this example, there are various issues to fix in hvPlot and HoloViews.

Reply:
Can we link to or at least list the issues involved?